.golangci.yml

```diff
@@ -0,0 +1,53 @@
+# options for analysis running
+run:
+  modules-download-mode: readonly
+  deadline: 10m
+  timeout: 10m
+  skip-files:
+    - ".*\\.pb.*\\.go"
```

Comment:
Can remove this since this repo won't be generating Go code from Protobuf definitions.

Reply:
👍